cost/turbonomics/buy_reserved_instances_recommendations/azure/turbonomics_buy_reserved_instances.pt

```diff
@@ -0,0 +1,304 @@
+              ri.effectiveCost = 0.0
+            }
+            ri.accountID=businessUnit.accountID
+            ri.accountName=businessUnit.accountName
+            ri.savings = (Math.round(ri.savings * 730 * 1000) / 1000)
+            ri.effectiveCost = (Math.round(ri.effectiveCost * 730 * 1000) / 1000)
+            ri.recurringCost = (Math.round(ri.recurringCost * 730 * 1000) / 1000)
+            monthlySavings = monthlySavings + ri.savings
+            instances.push(ri)
+          }
+        }
+      })
+    })
+
+    message = ""
+    if (instances.length != 0) {
+      pretty_savings = instances[0].savingsCurrency + ' ' + formatNumber(monthlySavings.toFixed(2), ",")
+      message = "The total estimated monthly savings are " + pretty_savings + '.'
+    }
+
+    result = {
+      'instances': instances,
+      'message': message
+    }
+EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_turbonomics_buy_ri" do
+  validate $ds_filtered_turbonomics_recommendations do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data.instances }} Turbonomic Purchase Reserved Instance Recommendations found"
+    detail_template <<-EOS
+## Turbonomics Buy RI
+{{data.message}}
+EOS
+    check eq(size(val(data, "instances")), 0)
+    export "instances" do
+      field "accountID" do
+        label "Account ID"
+      end
+      field "accountName" do
+        label "Account Name"
+      end
+      field "savingsCurrency" do
+        label "Savings Currency"
+      end
+      field "savings" do
+        label "Savings"
+      end
+      field "details" do
+        label "Recommendation Details"
+      end
+      field "numberOfInstances" do
+        label "Number of Reserved Instances"
+      end
+      field "resourceType" do
+        label "Resource Type"
+      end
+      field "term" do
+        label "Term"
+      end
+      field "region" do
+        label "Region"
+      end
+      field "service" do
+        label "Service"
+      end
+      field "upfrontCost" do
+        label "Up Front Cost"
+      end
+      field "recurringCost" do
+        label "Recurring Cost"
+      end
+      field "effectiveCost" do
+        label "Effective Cost"
+      end
+      field "sizeFlexible" do
+        label "Size Flexible"
+      end
+    end
+    escalate $esc_email
+  end
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "esc_email" do
+  automatic true
+  label "Send Email"
+  description "Send incident email"
+  email $param_email
+end
+
+###############################################################################
```

Comment:
I would remove cloud workflow comment since you don't have any cloud workflow, this is very relative, you don't have to do it :)

Reply:
Yes, I forgot to remove,